developer-tools.md

```diff
@@ -7,6 +7,31 @@ navigation:
   show: true
 ---
 
+<h2>Test coverage</h2>
+
+Apache Spark community uses various resources to maintain the community test coverage.
+
+<h3 id="github_action">GitHub Action</h3>
+
+[GitHub Action](https://github.com/apache/spark/actions) provides the following on Ubuntu 20.04.
+- Scala 2.12/2.13 SBT build with Java 8
+- Scala 2.12 Maven build with Java 11/17
+- Java/Scala/Python/R unit tests with Java 8/Scala 2.12/SBT
+- TPCDS ScaleFactor=1 Benchmark
```

Comment:
Should we maybe mention about Java version here too? It's a nit but I would call it like `TPC-DS benchmark with scale factor 1`